recipes/orange3/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=/dev/null
```

Comment:
Given that they both just use `setup.py` you can use the `script` tag in [build](http://conda.pydata.org/docs/building/meta-yaml.html#script)